### PR TITLE
package.json: Upgrade to React 15.1.0

### PR DESCRIPTION
**Pull request by ockham**
cc @rralian for review

### CODE REVIEW

**Comment by ockham** [comment]
Related: Automattic/wp-calypso#5116

File: package.json [hunk 1]
```diff
@@ -34,9 +34,9 @@
     "test": "test"
   },
   "dependencies": {
-    "react": "^0.14.3",
-    "react-addons-create-fragment": "^0.14.3",
-    "react-dom": "^0.14.3"
+    "react": "^15.1.0",
```
Comment (blowery) [inline package.json]: should these be ranges instead? it works with both 14 and 15, not just 15.

**Comment by ockham** [comment]
Like so? f1737d7

**Comment by blowery** [comment]
lovely. ship it

**Pull request merged**